<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BigHeadPictureWindow</class>
 <widget class="QMainWindow" name="BigHeadPictureWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Â§ßÂ§¥ÁÖßÁõ∏Êú∫</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="styleSheet">
       <string notr="true">font-size: 20px; font-weight: bold; padding: 10px;</string>
      </property>
      <property name="text">
       <string>ü§≥ Â§ßÂ§¥ÁÖßÁõ∏Êú∫</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="previewLayout">
      <item>
       <layout class="QVBoxLayout" name="cameraContainerLayout"/>
      </item>
      <item>
       <widget class="QLabel" name="labelBackground">
        <property name="styleSheet">
         <string notr="true">
                                        background-color: transparent;
                                        border: none;
                                        font-size: 16px;
                                        color: white;
                                    </string>
        </property>
        <property name="text">
         <string>ËÉåÊôØÈ¢ÑËßàÂå∫Âüü</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="cameraOverlayLayout"/>
      </item>
      <item>
       <widget class="QLabel" name="labelHint">
        <property name="styleSheet">
         <string notr="true">color: white; font-size: 14px; padding: 5px;</string>
        </property>
        <property name="text">
         <string>ËØ∑Â∞ÜÈù¢ÈÉ®ÂØπÂáÜÂúÜÂΩ¢Âå∫Âüü</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bgControlLayout">
      <item>
       <widget class="QPushButton" name="btnPrevBg">
        <property name="styleSheet">
         <string notr="true">
                                        QPushButton {
                                            background-color: #4A90E2;
                                            color: white;
                                            border-radius: 8px;
                                            font-size: 14px;
                                            font-weight: bold;
                                            margin: 5px;
                                            padding: 8px;
                                        }
                                        QPushButton:pressed {
                                            background-color: #3A80D2;
                                        }
                                        QPushButton:disabled {
                                            background-color: #CCCCCC;
                                        }
                                    </string>
        </property>
        <property name="text">
         <string>‚óÄ ‰∏ä‰∏ÄËÉåÊôØ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBgName">
        <property name="styleSheet">
         <string notr="true">
                                        font-size: 15px;
                                        font-weight: bold;
                                        padding: 10px;
                                        border: 1px solid #DDDDDD;
                                        border-radius: 5px;
                                    </string>
        </property>
        <property name="text">
         <string>ËÉåÊôØ1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNextBg">
        <property name="styleSheet">
         <string notr="true">
                                        QPushButton {
                                            background-color: #4A90E2;
                                            color: white;
                                            border-radius: 8px;
                                            font-size: 14px;
                                            font-weight: bold;
                                            margin: 5px;
                                            padding: 8px;
                                        }
                                        QPushButton:pressed {
                                            background-color: #3A80D2;
                                        }
                                        QPushButton:disabled {
                                            background-color: #CCCCCC;
                                        }
                                    </string>
        </property>
        <property name="text">
         <string>‰∏ã‰∏ÄËÉåÊôØ ‚ñ∂</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="cameraControlLayout">
      <item>
       <widget class="QPushButton" name="btnToggleCamera">
        <property name="styleSheet">
         <string notr="true">
                                        QPushButton {
                                            background-color: #34C759;
                                            color: white;
                                            border-radius: 8px;
                                            font-size: 16px;
                                            font-weight: bold;
                                            margin: 5px;
                                            padding: 10px;
                                        }
                                        QPushButton:pressed {
                                            background-color: #2EB750;
                                        }
                                    </string>
        </property>
        <property name="text">
         <string>üì∑ ÂºÄÂêØÁõ∏Êú∫</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCapture">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
                                        QPushButton {
                                            background-color: #FF3B30;
                                            color: white;
                                            border-radius: 8px;
                                            font-size: 16px;
                                            font-weight: bold;
                                            margin: 5px;
                                            padding: 10px;
                                        }
                                        QPushButton:pressed {
                                            background-color: #E0281E;
                                        }
                                        QPushButton:disabled {
                                            background-color: #CCCCCC;
                                        }
                                    </string>
        </property>
        <property name="text">
         <string>üì∏ ÊãçÁÖß‰øùÂ≠ò</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelStatus">
      <property name="styleSheet">
       <string notr="true">
                            color: #666666;
                            font-size: 13px;
                            padding: 8px;
                            border-top: 1px solid #eeeeee;
                            background-color: #f9f9f9;
                        </string>
      </property>
      <property name="text">
       <string>ÂáÜÂ§áÂ∞±Áª™</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
